core/app/models/spree/wished_product.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  class WishedProduct < Spree::Base
+    belongs_to :variant
+    belongs_to :wishlist
+
+    has_one :product, through: :variant
+
+    def total
+      quantity * (variant.price || 0)
```

Comment:
@MatthewKennedy and use `price_in` - we will drop the `price` method in 5.0 - https://github.com/spree/spree/issues/11243